# An array of poses with a header for global reference, and a time from start for each pose
Header header
geometry_msgs/Pose[] poses
duration[] time_from_start
